tests/integration/ref_indexer/compound-index.yml

```diff
@@ -0,0 +1,7 @@
+!Flow
+with:
+  show_exc_info: true
```

Comment:
I thought the `show_exc_info` goes in the pods only? or is the other way around?

Reply:
I think from here it may be passrd to pods but not sure